src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -188,7 +188,7 @@ public AzureAppConfigurationOptions SelectSnapshot(string name)
         }
 
         /// <summary>
-        /// Enables Azure App Configuration feature flags to be parsed and transformed into feature management configuration.
+        /// Configures options for Azure App Configuration feature flags that will be parsed and transformed into feature management configuration.
```

Comment:
We're missing a bit here.

* Selects feature flags
* Allows filtering of feature flags
* Set's up individual feature flag watching for selected feature flags.

Reply:
Should we mention the feature flag selecting/filtering even though it's summarized in `FeatureFlagOptions`? I think the watching isn't mentioned anywhere currently though.